test/modelValidatorTests.ts

```diff
@@ -96,19 +96,25 @@ describe("Model Validation", () => {
 
     it("should fail for paths with path parameter value resulting in duplicate forward slashes", async () => {
       const operationIds = "StorageAccounts_duplicateforwardslashes";
-      try {
-        const result = await validate.validateExamples(specPath, operationIds, {
-          consoleLogLevel: "off",
-        });
-        assert(
-          result.length !== 0,
-          `swagger "${specPath}" with operation "${operationIds}" contains passed incorrectly.`
-        );
-        // console.log(result)
-      } catch (err) {
-        assert.strictEqual(err.code, "REQUEST_VALIDATION_ERROR");
-        assert.strictEqual(err.innerErrors[0].code, "DOUBLE_FORWARD_SLASHES_IN_URL");
-      }
+      const result = await validate.validateExamples(specPath, operationIds, {
+        consoleLogLevel: "off",
+      });
+      assert(
+        result.length === 1,
+        `swagger "${specPath}" with operation "${operationIds}" contains passed incorrectly.`
```

Comment:
Wording of 'contains passed incorrectly' is confusing, can you change to 'contains validation errors'?